src/uu/stty/src/stty.rs

```diff
@@ -396,15 +412,25 @@ fn stty(opts: &Options) -> UResult<()> {
                     // combination setting
                     } else if let Some(combo) = string_to_combo(arg) {
                         valid_args.append(&mut combo_to_flags(combo));
+                    } else if arg.starts_with('-') {
```

Comment:
Mhlhl, this implies that doing `stty foo bar` would pass, until we fail parsing... While to me we should just fail as GNU does in that case with `stty: invalid argument 'foo'`